src/encoded/static/components/facets/defaults.js

```diff
@@ -1015,15 +1066,59 @@ export const DefaultFacet = ({ facet, results, mode, relevantFilters, pathname,
         setTypeaheadTerm(event.target.value);
     };
 
+    /**
+     * Handlers opening or closing a tab
+     *
+     * @param {event} e React synthetic event
+     * @param {bool} status True for open, false for closed
+     * @param {string} field Tab name
+     */
+    const handleExpanderClick = (e, status, field) => {
+        if (toggleFacets) {
+            toggleFacets(e, status, field);
+        }
+    };
+
+    /**
+    * Called when user types a key while focused on a facet term. If the user types a space or
+    * return we call the term click handler -- needed for a11y because we have a <div> acting as a
+    * button instead of an actual <button>.
+    *
+    * @param {event} e React synthetic event
+    * @param {bool} status True for open, false for closed
+    * @param {string} field Tab name
+    */
+    const handleKeyDown = (e, status, field) => {
+        if (e.keyCode === 13 || e.keyCode === 32) {
+            // keyCode: 13 = enter-key. 32 = spacebar
+            e.preventDefault();
+            handleExpanderClick(e, status, field);
+        }
+    };
```

Comment:
I think this and `handleExpanderClick` should either be a service provided by search.js, or should be in a custom hook that all facet types can use. Or maybe just be implemented once in TitleComponent.